maquetado/index.html

```diff
@@ -10,6 +10,10 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
+  <div class="navbar">
+    <img class="navbar-logo" src="./assets/wolox_logo.svg">
```

Comment:
Remember to add the `alt` attribute to your images. Good work with the naming here.